components/dropdown/README.md

```diff
@@ -176,6 +178,18 @@ dropdown.addEventListener('click', function() {
 * `d2l-dropdown-open`: dispatched when the dropdown is opened
 * `d2l-dropdown-close`: dispatched when the dropdown is closed
 
+**Slots:**
```

Comment:
This might be slightly confusing or misleading since there are three slots - it might be worth mentioning the default slot. In other places, I didn't both listing the slots since the snippet already shows them.  🤷‍♂

Reply:
Is it more or less confusing? I mean anything that isn't in a named slot ends up in the default content slot, which has no name and you don't make a div for 🤔 Might just be me

Reply:
Possibly.  I guess the other thing is being consistent about whether we document the slots this way.  So far I've been opting for less text especially when the snippet already conveys the info.  I don't feel super strongly about it, but I'd rather be consistent either way.

Reply:
I'm wondering if having some explicit listing like this might be useful in places where the slots are functionally optional, or if that's easy to assume from the outside? I haven't seen enough cases to know.

If we don't need them explicitly listed I can make the adjustment in another PR and check the docs in general for that consistency with that. 🙂 Just wanted to see what thoughts were (but merging this in for now so I can press on with other things in the meantime)